README.md

```diff
@@ -107,6 +107,8 @@ First, see if the answer to your question can be found in the [Guides] or the
 the [Tokio Gitter channel][chat]. We would be happy to try to answer your
 question.  Last, if that doesn't work, try opening an [issue] with the question.
 
+[Guides]: https://tokio.rs/docs/getting-started/hello-world/
```

Comment:
```suggestion
[Guides]: https://tokio.rs/docs
```

/docs should redirect as needed.

Reply:
Should we also update the other links to the Guides?

```
$ rg -n 'tokio.rs/docs/getting-started/hello-world'
README.md
30:[Guides](https://tokio.rs/docs/getting-started/hello-world/) |
110: the suggested change

tokio/README.md
30:[Guides](https://tokio.rs/docs/getting-started/hello-world/) |

tokio/src/lib.rs
24://! [website]: https://tokio.rs/docs/getting-started/hello-world/
```

Reply:
Probably! `/docs` should be more reliable as an entry point.